kubernetes.yml

```diff
@@ -27,8 +27,8 @@ metadata:
   labels:
     app: app-serv
 spec:
-  type: 
-    NodePort
+  # type: 
+  #   NodePort
```

Comment:
Did you mean to leave this out?

Reply:
Yes, according to lecture material we should avoid using NodePort and we should ClusterIP with ingress which was already implemented anywa. Just forgot to delete this part, but next pr can do this.